### ISSUE TITLE
[Bug]: stream gets stopped when app goes to background state

### ISSUE THREAD

**Issue by talhatsk** [apivideo/api.video-reactnative-live-stream#88]
### Version

v2.0.0

### Which operating systems have you used?

- [X] Android
- [ ] iOS

### Environment that reproduces the issue

Oneplus 8T
android 13

### Is it reproducible in the example application?

Yes

### RTMP Server

*

### Reproduction steps

1. start the stream 
2. take the app to background 
3. stream gets stopped 

### Expected result

on going to background state, the stream should not stop

### Actual result

the stream gets stopped as soon as user takes the app to background

**Comment by ThibaultBee** [comment]
Hi,
It is on purpose. 
It is not clear if camera and microphone could be used in background in a foreground service. What is clear is that it is not possible for older Android version.

There are multiple topics on stackoverflow about this: https://stackoverflow.com/questions/67284842/camera2-cameramanager-opencamera-exception-from-time-to-time-on-some-devices

**Comment by talhatsk** [comment]
> Hi, It is on purpose. It is not clear if camera and microphone could be used in background in a foreground service. What is clear is that it is not possible for older Android version.
> 
> There are multiple topics on stackoverflow about this: https://stackoverflow.com/questions/67284842/camera2-cameramanager-opencamera-exception-from-time-to-time-on-some-devices

I'm intended to make a Picture in Picture feature for android, for that i guess camera and microphone will work in background. Is there any way to do now?

**Comment by ThibaultBee** [comment]
There is no background support for `@api.video/react-native-livestream`.

**Comment by talhatsk** [comment]
> There is no background support for `@api.video/react-native-livestream`.

are you planning to do it in future?

**Comment by ThibaultBee** [comment]
Maybe. Just not right now.
Are you 100% sure you need background for PiP?

**Comment by talhatsk** [comment]
> Maybe. Just not right now. Are you 100% sure you need background for PiP?

yes when the app is put to background state a PiP should appear showing the stream, that's the requiremnet.

**Comment by ThibaultBee** [comment]
Could you provide a sample so we can test this quickly?

**Comment by talhatsk** [comment]
> Could you provide a sample so we can test this quickly?

Should i create a new project and provide you a sample for it by integrating `PiP` with `@api.video/react-native-livestream`?
